<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimUI</class>
 <widget class="QWidget" name="SimUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sim</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,2">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="hbox_above" stretch="1,1,8,1,2">
     <item>
      <widget class="QComboBox" name="cBox_mode">
       <item>
        <property name="text">
         <string>offLine</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>inLine</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_data">
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_data">
       <property name="text">
        <string>Pls select a data source</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lab_coordMouse">
       <property name="text">
        <string>Coordinate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_mid">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="lab_image">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWChartView" name="qcharts_data" native="true"/>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gBox_bottom"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWChartView</class>
   <extends>QWidget</extends>
   <header location="global">qwchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="sim.qrc"/>
 </resources>
 <connections/>
</ui>
